charts/capsule/templates/post-install-job.yaml

```diff
@@ -28,7 +28,8 @@ spec:
       restartPolicy: Never
       containers:
       - name: post-install-job
-        image: "bitnami/kubectl:1.18"
+        image: {{ .Values.jobs.image.repository }}:{{ .Values.jobs.image.tag }}
```

Comment:
What about using a template function as we did with the manager container?

https://github.com/clastix/capsule/blob/4f34483dee32bcd8ff88c7590f1ab8e5cdcdfa9e/charts/capsule/templates/_helpers.tpl#L64:L69

Reply:
Great!

I'll adapt the `kube-rbac-proxy` as well in order to make manage all images the same way.